tools/envoy_headersplit/README.md

```diff
@@ -0,0 +1,13 @@
+# Envoy Header Split
+Tool for spliting monolith header files in envoy to speed up compilation
+
+
+Steps to divide Envoy mock headers:
+
+1. run `headersplit.py` to divide the monolith mock header into different classes
+
+2. Resolve bazel dependency for the divided classes manually and remove unused includes for them
```

Comment:
Can we clarify this bullet point? I'm not sure what it means; my guesses are:

-  Add new Bazel dependencies to the monolithic dependency
-  Write the new Bazel dependencies
-  Something else?

Reply:
The second one. After running `headersplit.py`, we will get some new mock class files. We need to write Bazel dependencies for them.
And since those new mock class file has the same #includes as the monolithic mock header, we need to clean up unused includes for them.
After refactoring the origin monolithic mock header is no longer used in anywhere since all of its occurrences will be replaced by the new mock class. Thus we don't need to add new Bazel dependencies to the monolithic header.
I will clarify it in README